docs/pages-for-subheaders/rke1-hardening-guide.md

```diff
@@ -12,10 +12,11 @@ This hardening guide is intended to be used for RKE clusters and is associated w
 
 | Rancher Version | CIS Benchmark Version | Kubernetes Version           |
 |-----------------|-----------------------|------------------------------|
-| Rancher v2.7    | Benchmark v1.23       | Kubernetes v1.23 up to v1.25 |
+| Rancher v2.7    | Benchmark v1.7       | Kubernetes v1.25 |
```

Comment:
Not sure if this is intentional, but aren't these the same changes from your RKE1 PR? 
https://github.com/rancher/rancher-docs/pull/794/files

I'm ok with the changes being in one PR, but if you'd like to keep them separate I think you should undo this change and leave it to the other PR.